src/webserver/Firebasey.jl

```diff
@@ -638,6 +638,9 @@ function direct_diff(old::Cell, new::Cell)
 	if old.folded ≠ new.folded
 		push!(changes, ReplacePatch([:folded], new.folded))
 	end
+	if old.isolated ≠ new.isolated
+		push!(changes, ReplacePatch([:isolated], new.isolated))
+	end
```

Comment:
This is actually skipped as script in the notebook, so it's not (I guess) used within Pluto but just for testing in the Firebasey notebook. Correct?